### DISCUSSION TITLE
X/Y/Z Plotting like in automatic

### DISCUSSION THREAD

**Discussion by TwoPintOhh** [Comfy-Org/ComfyUI#3907] [Q&A]
Hi. Does anyone know how to do X/Y/Z plots in comfy? more to the point i would like to select a bunch of lora's, a range of weights for the lora's, and promt searh and replace. this way i can let it run and test my loras with different prompts and get a bunch of X/Y grids.
I have looked around for hours and can't seem to find any example workflows for this. any pointers would be great. Thanks all

**Comment by 0xf1234** [comment]
Also trying to figure this out.

**Comment by mcmonkey4eva** [comment]
Swarm https://github.com/mcmonkeyprojects/SwarmUI lets you do grids of comfy workflows easily

**Comment by geroldmeisinger** [comment]
I've implemented a node suite which lets you do this in the most ComfyUI-way possible. It uses the native data lists feature, and doesn't need any weird samplers or black magic:
<img width="2390" height="789" alt="XYZ-GridPlot" src="https://github.com/user-attachments/assets/9d0e9479-7881-4dcd-948b-8b2dd308fa3f" />

https://github.com/geroldmeisinger/ComfyUI-outputlists-combiner#xyz-gridplot-simple
